scalability_and_performance/using-argocd.adoc

```diff
@@ -0,0 +1,61 @@
+[id="using-argocd"]
+= Using ArgoCD to manage cluster configuration
+include::modules/common-attributes.adoc[]
+:context: using-argocd
+
+You can use ArgoCD to implement a GitOps configuration management system with {product-title}. You can use ArgoCD to manage many {product-title} configuration files across multiple clusters.
+
+The following sections describe how to install ArgoCD and use configuration files from a samples repository on GitHub to apply different configuration files to a cluster and a build configuration file to multiple clusters.
+
+Before you complete these processes, you might want to learn more about ArgoCD and the way it uses kustomize to modify the configuration files for different clusters. See link:https://argoproj.github.io/argo-cd/[Argo CD - Declarative GitOps CD for Kubernetes], the ArgoCD documentation, and link:https://kustomize.io/[Kustomize] for more information.
+
+toc::[]
+
+include::modules/argocd-gitops-overview.adoc[leveloffset=+1]
+
+include::modules/argocd-overview.adoc[leveloffset=+1]
+
+include::modules/argocd-installing-cli.adoc[leveloffset=+1]
+
+include::modules/argocd-installing-argocd.adoc[leveloffset=+1]
+
+include::modules/argocd-forking-sample-repository.adoc[leveloffset=+1]
+
+include::modules/argocd-adding-sample-repository.adoc[leveloffset=+1]
+
+[id="using-argocd-manage-applications"]
+== Using ArgoCD to manage applications
+
+You can use ArgoCD _applications_ to deliver global custom resources such as those that configure {product-title} clusters. Each application requires a namespace, but you can provide the default namespace for applications that deliver global resources.
+
+You can use ArgoCD to deliver many of the xref:../installing/install_config/customizations.adoc#customizations[Available cluster customizations].
+
+[NOTE]
+====
+By default, ArgoCD attempts to prune resources if you delete the application that delivered them. Because {product-title} often blocks the deletion of global configuration custom resources, ArgoCD can become blocked if it cannot prune resource as expected. Because of this behavior, include the  `argocd.argoproj.io/sync-options: Prune=false` annotation in the custom resources that you define for ArgoCD.
+// If you do run into this problem, you will need to manually "kubectl edit" the Argo Application and remove the finalizer which blocks until resources are pruned.
```

Comment:
I think this part should be added to NOTE section, because `argocd.argoproj.io/sync-options: Prune=false` annotation might not be useful when Argocd tries to delete a resource that can't allow to be deleted, so I think we can add a solution for this situation.